vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```diff
@@ -0,0 +1,57 @@
+.dialog {
+}
+
+.dialog::part(overlay) {
+  width: 66vw;
+}
+
+.dialog .confirm-button {
+  background-color: var(--lumo-primary-color);
+  color: var(--lumo-primary-contrast-color);
+  font-weight: 600;
+  min-width: calc(var(--lumo-button-size) * 2.5);
+}
+
+.dialog .confirm-button:hover::before {
+  opacity: 0.05;
+}
+
+.create-project-dialog {
+
+}
+
+.create-project-dialog .project-form-layout {
+  max-width: 60vw;
+  padding-bottom: 1em;
+}
+
+.create-project-dialog .code-and-title {
+  align-items: baseline;
+  width: 100%;
+}
+
+.create-project-dialog .code-and-title .code {
+  max-width: 20vw;
+}
+
+.create-project-dialog .code-and-title .title {
+  width: 100%;
+}
+
+.create-project-dialog .search {
+  max-width: 30vw;
+}
+
+.create-project-dialog .project-contacts {
+  margin: 0;
+  padding: var(--lumo-space-m) 0 0;
+}
+
+.create-project-dialog .project-contacts .title {
+  font-weight: bold;
+}
+
+.create-project-form vaadin-form-item::part(label) {
+  font-weight: bold;
+  color: black;
+}
```

Comment:
Is this a request? It looks very "bold" I would argue 🤔 
![Bildschirmfoto 2023-06-20 um 13 27 44](https://github.com/qbicsoftware/data-manager-app/assets/29627977/b572aebd-329d-4884-b07c-58ac0f8ba5af)